perfdash/config/builds.json

```diff
@@ -0,0 +1,3 @@
+{
+	"gce-100Nodes": 100
```

Comment:
Like I mentioned earlier - let's not try to hardcode this config into perfdash (as then we'll need to rebuild and push everytime we want to change this).

Instead, let's pick this config from https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml (from where we're already picking perfDashTag). You can add a new label there like `perfDashRange` or sth - and we can easily change this range then by just a single PR changing that value.

Reply:
Good idea. I will change my code.

Reply:
Done. `perfDashBuilds` is a new label for specifying builds.